src/http/api/v1/matches.ts

```diff
@@ -111,5 +113,61 @@ export const createHandler = (config: HttpApiConfig): express.Router => {
     res.send(responseBody);
   }
 
+  router.post("/chains/:chainId/rounds/:roundId/estimate", async (req, res) => {
+    await estimateMatchesHandler(req, res, 200);
+  });
+
+  async function estimateMatchesHandler(
+    req: Request,
+    res: Response,
+    okStatusCode: number
+  ) {
+    const chainId = Number(req.params.chainId);
+    const roundId = req.params.roundId;
+    const potentialVotes = potentialVotesSchema
+      .parse((req.body as { potentialVotes: PotentialVotes }).potentialVotes)
+      .map((vote) => ({
+        ...vote,
+        amount: BigInt(vote.amount),
```

Comment:
Is this still necessary after using `coerce.bigint()` in the zod schema?

Reply:
Nope - good catch. I see that I'm rusty on zod heh.